source/scss/components/_multioffer-teaser.scss

```diff
@@ -110,6 +123,7 @@
 
   h4 {
     hyphens: auto;
```

Comment:
You may want to try out making the width 100%. The html-ellipsis fills itself, which is why the h4:empty does not trigger anymore.
<img width="373" alt="Screenshot 2019-07-17 at 10 40 47" src="https://user-images.githubusercontent.com/4340405/61360758-5ca2d900-a87f-11e9-96c9-d09765fb1a98.png">

Reply:
At least in the p-tag cases it works. Because there flex-width is 100%.

Reply:
working on improving the whole cpr thing